horovod/spark/task/task_info.py

```diff
@@ -23,7 +24,7 @@ def __init__(self):
 
 
 def get_available_devices():
-    if 'gpu' not in _info.resources:
+    if 'gpu' not in _info.resources or os.getenv('HOROVOD_USE_DEFAULT_GPU_INDEX') == '1':
```

Comment:
This env var has to be added to documentation.

Reply:
Docs and CHANGLOG are updated.